bitnami/dataplatform-bp2/templates/NOTES.txt

```diff
@@ -0,0 +1,39 @@
+** Data Platform Blueprint 2 is being deployed, it could take some time to be ready **
+
+The following components are being deployed to your cluster:
+
+{{- if .Values.kafka.enabled }}
+
+***********
+** Kafka **
+***********
+
+To access the Kafka service from your local machine execute the following:
+
+   kubectl port-forward --namespace {{ .Release.Namespace }} svc/{{ include "subcomponent.service.name" ( dict "componentName" "kafka" "context" $ ) }} 9092:9092 &
+   echo "Kafka service available at : http://127.0.0.1:9092"
+{{- end -}}
+
+{{- if .Values.elasticsearch.enabled }}
+
+**********
+** Elasticsearch **
+**********
+
+To access the Elasticsearch service from your local machine execute the following:
+
+   kubectl port-forward --namespace {{ .Release.Namespace }} svc/{{ include "subcomponent.service.name" ( dict "componentName" "coordinating-only" "context" $ ) }} 9200:9200 &
+   echo "Elasticsearch service available at : http://127.0.0.1:9200"
+{{- end -}}
+
+{{- if .Values.spark.enabled }}
+
+***********
+** Spark **
+***********
+
+To access the Spark service from your local machine execute the following:
+
+   kubectl port-forward --namespace {{ .Release.Namespace }} svc/{{ include "subcomponent.service.name" ( dict "componentName" "spark" "context" $ ) }} 8080:80 &
+   echo "Spark service available at : http://127.0.0.1:8080"
+{{- end -}}
```

Comment:
You're suggestion to use a "port-forward" approach regardless the service type used on each subchart to expose the application. Is this intended?

Reply:
I guess this was copied from BP1 but I will add the LoadBalancer too